ui/components/component-library/avatar-network/avatar-network.tsx

```diff
@@ -21,8 +20,7 @@ export const AvatarNetwork: AvatarNetworkComponent = React.forwardRef(
       src,
       showHalo,
       color = TextColor.textDefault,
-      backgroundColor = BackgroundColor.backgroundAlternative,
-      borderColor = BorderColor.transparent,
```

Comment:
Can you explain the removal of `borderColor`?  Since this modifies something in the component library, it will affect places using `AvatarNetwork` beyond just the token list.  And some of those places pass a `borderColor`.

Reply:
Yes, this was actually Amanda's direction. The border was making the icon 2px smaller than was intended, since `border` technically takes up space within an element: https://www.thoughtco.com/css-outline-styles-3466217

So this was throwing off some of the visual polish of the UI. These are the places that I focused on:

1. Token List
2. Import Tokens Modal
3. Add Network Settings View
4. Add Network Modal

Amanda did suggest that I follow up with George Marshall regarding this after P0. Do you want me to do another check to see if I missed any `AvatarNetwork`?

Reply:
I'm also noticing some code related to `showHalo` which seems like it might be some tech debt to be cleaned up at some point.

Reply:
Okay just making sure we want to affect all the `AvatarNetwork`.  I do see them get 2px bigger so good that it's intended.  Would be good to audit all instances of `AvatarNetwork`, specifically ones explicitly passing a `borderColor` to make sure they match the new visual.

https://github.com/user-attachments/assets/72d47c9d-c509-45bb-a685-c1cefa006a79

Reply:
Will request George Marshall on this PR and follow up with him after P0

Reply:
Thanks for the detailed explanation, @gambinish ! The change to use an outline instead of a border to avoid the 2px reduction in avatar size makes sense for maintaining visual consistency. However, I wanted to highlight a couple of points:

1. **Overrides of `borderColor`**: (Also mentioned by @bergeron) With this change, any instances of `AvatarNetwork` that currently rely on the `borderColor` prop for customization will now require engineers to use SCSS to apply the outline. This could complicate the styling process and make it less intuitive for engineers who are accustomed to using utility props.

2. **Accessibility Implications**: Outlines are often used to indicate focus states, which are essential for keyboard navigation and accessibility. Using outlines for purely decorative purposes might conflict with accessibility best practices. We need to ensure that this change does not interfere with focus indicators or the overall accessibility of the component. It's unlikely that we would want to focus on a singular Avatar but not out of the realm of possibilities. It might be worth exploring alternative solutions that do not compromise accessibility.